capstone/src/test/java/com/google/step/clients/RestaurantClientTest.java

```diff
@@ -112,18 +93,21 @@ public void testGetSingleRestaurant() {
 
     // Run client method
     RestaurantClient restaurantClient = new RestaurantClient();
-    Restaurant result = restaurantClient.getSingleRestaurant(restaurantKey);
+    Optional<Restaurant> result = restaurantClient.getSingleRestaurant(restaurantKey);
+    Restaurant actual = null;
+    if (result.isPresent()) {
```

Comment:
I would just do `assertTrue(result.isPresent());`, and then assume optional.get works as intended (and if it somehow doesn't, you'd still catch it with the comparisons later).